src/Menu.java

```diff
@@ -0,0 +1,156 @@
+import org.json.simple.JSONArray;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void main(String[] args){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                //Utils.println("starting_menu",1,2,3,4, Game.isSoundOn()? "Off": "On",5,6);
+                JSONArray menu = Settings.getArray("starting_menu");
+                for(int i = 0; i < menu.size(); i++) {
+                    Utils.println("display_shop_item", i + 1, menu.get(i), "");
+                }
+                    userInput = Integer.parseInt(reader.readLine());
+
+                switch(userInput){
+                    case(1):
+                        Game.play();
+                        break;
+                    case(2):
+                        //Learn about the trail
+                        learnAboutTrail();
+                        break;
+                    case(3):
+                        //See the top 10
+                        oregonTopTen();
+                        break;
+                    case(4):
+                        //Turn sound off
+                        Game.setSoundOn(!Game.isSoundOn());
```

Comment:
This message is still not shown after toggling the sound, you need to add the `nextLine()` method here too. Also you should avoid using `System.out.println()`. The `Utils.println()` method was created to have the strings in a json file instead of the code. It might be easier if you create a new method `promptln()` (and `prompt()`) that calls `Utils.println()` and `Keyboard.nextLine()` for you.